### ISSUE TITLE
Attached vibrator device outputting 0.1 when it should be 0

### ISSUE THREAD

**Issue by sextoydb** [FredTungsten/ScriptPlayer#64]
I noticed that scriptplayer haptics output was quite different to syncydink, namely, the Lush I have connected via buttplug never stops vibrating! I'm not sure if this was intended feature, but it feels like a bug. Specifically, in funscripts with a 0 position, it sends a 0.1 vibrate command to buttplug, instead of 0. 

I found the line I think is responsible:

```
public static double LaunchToVibrator(byte position)
        {
            const double max = 1.0;
            const double min = 0.1;

            double speedRelative = 1.0 - ((position + 1) / 100.0);
            double result = min + (max - min) * speedRelative;
            return Math.Min(max, Math.Max(min, result));
        }
```

Any chance we could change the actual minimum to be 0, or some kind of toggle in settings?

Thanks!

**Comment by FredTungsten** [comment]
I think that was on prupose but I can't quite remember the reason (I think some vibrators didn't do anything for values < 0.1). Since there are separate min/max settings in the UI now, I think I'll just set the minimum to 0 again.

**Comment by sextoydb** [comment]
Thank you!

**Comment by FredTungsten** [comment]
Fixed in commit c29da8b09817c412467015b0c919e36ac3a42ca0, [build 1.0.5.120](https://ci.appveyor.com/project/FredTungsten/scriptplayer/build/1.0.5.120/job/m15ci8aw57v34lgu/artifacts)
